src/services/userService.js

```diff
@@ -246,7 +256,15 @@ async function allAdmins() {
     return allRegular;
 }
 
+async function allUsers() {
+    const allUsers = await User.find();
```

Comment:
What about deleted users? Currently you are returning only not deleted organizers, but all users. Be consistent. If you want, ypou can pass a parameter where the front end can explicitly pass a param if they need all users/organizers, etc. even the deleted. If not passed, return only not deleted. And it must be consistent.